inst/tests/tests.Rraw

```diff
@@ -18286,3 +18286,17 @@ test(2246.1, DT[, data.table::shift(b), by=a], DT[, shift(b), by=a], output="GFo
 test(2246.2, DT[, data.table::first(b), by=a], DT[, first(b), by=a], output="GForce TRUE")
 test(2246.3, DT[, data.table::last(b), by=a], DT[, last(b), by=a], output="GForce TRUE")
 options(old)
+
+# unique.data.table doesn't return a mutable alias, #5960
+# unique nrows > 1 should return different address object (even before this fix)
+DT1 = data.table(a = 1:2, b = 3:4)
```

Comment:
Isn't this test essentially doing the same as test 1861?

Reply:
I would move this test to be grouped there, since it's really the same behavior we're testing.